docs/contributing/triaging.md

```diff
@@ -0,0 +1,141 @@
+
+```
+https://github.com/username/repository/labels/label-name
+```
+
+### Example
+
+For example, to link to the "good first issue" label in the nsidc/earthaccess repository, you would use the URL:
+
+```
+https://github.com/nsidc/earthaccess/labels/good%20first%20issue
+```
+
+
+## Close Issue as Not Planned
+
+This section will cover the guidelines for when to use the "Close Issue as Not Planned" feature, and how to handle issues that are not planned or feasible.
+
+### When to "Close as not planned"?
+
+Close issues as "not planned" when:
+
+- An issue is not aligned with the project's goals or priorities.
+- An issue is not feasible to be addressed due to technical or resource constraints.
+- An issue is a duplicate of an existing issue that has already been addressed.
+
+When closing an issue as not planned, keep the following best practices in mind:
+
+- Provide a clear explanation as to why the issue is not planned or feasible.
+- Offer alternative solutions or workarounds, if possible.
+- Link to relevant documentation or resources, if applicable.
+
+## Discussions vs Issues
+
+This section would cover the guidelines for when to use discussions versus issues, and how to migrate between them.
+
+###  What are Discussions?
+
+Discussions are used for:
+
+- Brainstorming and idea generation.
+- Project feedback.
+- General questions and topics.
+
+###  What are Issues?
+
+Issues are used for:
+
+- Reporting bugs and errors.
+- Tracking progress on specific tasks or projects.
+- Requesting changes or improvements.
+
+### When to Migrate
+
+Use your best judgement when migrating between issues and discussions. Sometimes it makes more sense to open a new issue or discussion instead of migrating, for example when there are many things being discussed, but we want to create an issue or task out of just one of those things.
+
+Migrate a discussion to an issue when:
+
+- A specific task is identified.
+- A bug or error is reported.
+- A change or improvement is requested.
+
+Migrate an issue to a discussion when:
+
+- The issue is a nebulous idea that needs to be workshopped before it can be implemented.
+- The issue is a general question or topic.
+- The issue is not specific or actionable.
+
+## Issue Triaging Workflow
+
+``` mermaid
+
+flowchart TD
+  %%{init: {"flowchart": {"htmlLabels": false}} }%%
+  classDef default font-size:32pt;
+  start{"`Followed
+  issue
+  template?`"}
+  start ==NO==> close1[Change
+  to draft and ask to follow template]
+  start == YES ==> dupe{Is duplicate?}
+  dupe == YES ==> close2[Close and point to duplicate]
+  dupe == NO ==> repro{Has proper reproduction?}
+  repro == NO ==> close3[Label: 'needs reproduction' bot will auto close if no update has been made in 3 days]
```

Comment:
>bot will auto close if no update has been made in 3 days

I think this isn't true. We have one automation for autoclosing issues, but I believe it's broken: https://github.com/nsidc/earthaccess/blob/main/.github/workflows/issue-manager.yml

We should be careful about fixing it: #895 & https://jacobtomlinson.dev/posts/2024/most-stale-bots-are-anti-user-and-anti-contributor-but-they-dont-have-to-be/

Reply:
Ah, that makes sense, thanks for the context!

Reply:
I think getting @asteiker 's eyes on this is probably the best way forward. Will you be at next week's hack day? Maybe we can work together to finalize the flowchart.

Reply:
@mfisher87 @Sherwin-14 Apologies for my radio silence on this recently! Yes I am happy to get back into this and work together at the next hackday. I'll review this in more detail in preparation.

Reply:
Thanks, Amy!